cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
I stand by my point that hostfolders are defined on an instance level. This especially makes sense when `minishift create` is considered.

To me, an instance can represent a set of work and services that might have different storage needs.

I originally placed it in the global config. But when more than one instance is considered, why would I want to mount my homefolder in all of them?

It might be me... but I do understand the convenience aspect. I just wonder how often this will be the case.